pkg/odo/cli/catalog/describe/service.go

```diff
@@ -42,20 +62,88 @@ func NewDescribeServiceOptions() *DescribeServiceOptions {
 
 // Complete completes DescribeServiceOptions after they've been created
 func (o *DescribeServiceOptions) Complete(name string, cmd *cobra.Command, args []string) (err error) {
+	isSVCSupported, err := svc.IsCSVSupported()
+	if err != nil {
+		isSVCSupported = false
+	}
+	if strings.Contains(args[0], "/") && isSVCSupported {
```

Comment:
In place of:
```go
strings.Contains(args[0], "/")
```
please try to use https://github.com/openshift/odo/blob/3b11f9d5612f9ee26494748ad8fa926d7b79fbd6/pkg/service/service.go#L564

I'm requesting this bceause we use this function at other places to differentiate between Operators and Service Catalog.

Also, `strings.Contains(args[0], "/")` would pass if someone did `odo catalog describe service operator-name/` without specifying the CR name after `/`.

Reply:
yeah, I kind off did that on purpose because that check which you mentioned actually seems like a validation and hence is added in the validation section https://github.com/openshift/odo/pull/4671/files#diff-bb8c7d224895f090c5fc3f1c166deb273b1f1d47c6e8d5e4af8cdf94cc9957d6R88

Also I already went through that function and it would correctly but name of function suggests its to split Service Kind and Service Name ( i.e. the instanciated operand ). I am spliting operator and cr type. So I thought it would be confusing to use

Reply:
> but name of function suggests its to split Service Kind and Service Name ( i.e. the instanciated operand ). I am spliting operator and cr type. So I thought it would be confusing to use

Yeah, that's my bad. But we've successfully been using it at various places for both `<operator>/<CR>` and `<CR>/<service-name>` checks.

> seems like a validation and hence is added in the validation section

One way to look at it is it helps us decide if the input entered by the user on CLI is Operator backed or not. That seems like a good thing to be in `Complete`, IMO.

Reply:
ok

Reply:
used